README.md

```diff
@@ -66,6 +66,9 @@ are not allowed!):
 More details about RAJA configuration options are located in the User 
 Documentation.
 
+We also maintain a [**RAJA Template Project**](https://github.com/LLNL/RAJA-project-template) that shows how to use RAJA in a CMake project, either as a Git
```

Comment:
👍

Reply:
I thought you'd like that....

Reply:
I am a fan of the RAJA template project.

Reply:
You're going to give @davidbeckingsale an ego problem.... :-)